src/adapters/gam/managers/orders.py

```diff
@@ -211,7 +294,7 @@ def get_order_line_items(self, order_id: str) -> list[dict]:
             statement = statement_builder.ToStatement()
 
             result = lineitem_service.getLineItemsByStatement(statement)
-            return result.get("results", [])
+            return result.get("results", []) if isinstance(result, dict) else getattr(result, "results", [])
```

Comment:
This is kind of wacky. We don't know what kind of object it is? This kind of goes to the MyPy point. We should be passing around objects that we know the structure of.